test/integration/api-schema/_utils.js

```diff
@@ -5,19 +5,20 @@ const fs = require('fs')
 const tmpdir = require('os').tmpdir
 const join = require('path').join
 
-const validator = require('is-my-json-valid')
-const refParser = require('@apidevtools/json-schema-ref-parser')
+const Ajv = require('ajv')
 const rimraf = require('rimraf')
 const thunky = require('thunky')
 
+const ajv = new Ajv({ allErrors: true })
+
 const schemaDir = thunky(function (cb) {
-  const dir = join(tmpdir(), '.schemacache')
+  const dir = join(tmpdir(), 'apm-schema-v2-cache')
```

Comment:
Note: I changed this to avoid those with an existing `.schemacache` dir accidentally still testing with the old schema files.

Also I changed away from a dot-file hidden dir because it seems unnecessary in a system tmpdir. The usage of ".schemacache" comes from apm-agent-python.git which puts the temp dir inside the git clone (at "./test/.schemacache").